doxydir/1_usage.md

```diff
@@ -19,6 +19,8 @@ ctx = japi_init(NULL);
 japi_register_request(ctx,"function_name",&function_handler);
 \endcode
 
+Note, that "function_name" may not start with "japi_" which is used to mark internal commands.
+
 The function will be saved in the passed JAPI context. The server which manages the requests, is started with:
 \code
```

Comment:
The Preview of these codeblocks does not work.
![grafik](https://github.com/Fraunhofer-IIS/libjapi/assets/139216767/b07e3353-b883-4ea9-b8c9-e5cf0a069b09)

I think this can be done for example like that. 
```
japi_context *ctx;
ctx = japi_init(NULL);
```

Reply:
Start codeblocks with "```" and end with "```"

Reply:
https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks

Reply:
Or is this look wanted?

Reply:
The markup is likely intended for the Doxygen documentation, and not for Github-flavoured Markdown. Are Markdown code fences also supported by Doxygen Markdown? If so, then those blocks should be refactored from `\code ... \endcode` to using fenced code blocks `'''C ... '''`

Reply:
Exactly. There is no shared syntax for hightlighted code betweeen Doxygen and GitLab flavoured Markdown. As the files in question are located in a directory called "doxydir", I think it's fine to not change it